packages/block-editor/src/store/private-selectors.js

```diff
@@ -139,3 +139,13 @@ export function getRemovalPromptData( state ) {
 export function getBlockRemovalRules( state ) {
 	return state.blockRemovalRules;
 }
+
+/**
+ * Returns the client ID of the block settings menu that is currently open.
+ *
+ * @param {Object} state Global application state.
+ * @return {string|undefined} The client ID of the block menu that is currently open.
```

Comment:
Did you mean `null` or `undefined`? The default value is `null` and currently we'll return `undefined` if a `clientId` is missing for some reason when dispatching the `setOpenedBlockSettingsMenu()` action (this feels like an inconsistent behavior on its own, so I'm commenting with it separately).